Formula/libtorrent-rasterbar.rb

```diff
@@ -1,8 +1,8 @@
 class LibtorrentRasterbar < Formula
```

Comment:
Eh, sorry I spoke too soon. Do you have anymore insight into what's up with why Rasterbar was removed? Does this formula need to be renamed?